docs/pages/distribution/publishing-websites.mdx

```diff
@@ -55,20 +119,28 @@ Use [Serve CLI](https://www.npmjs.com/package/serve) to quickly test locally how
 
 <Tabs>
 
-<Tab label="Expo Router">
+<Tab label="single">
 
-If you're using [static rendering](/router/reference/static-rendering), then you can serve the static bundle with the following command:
+<Terminal cmd={['$ npx serve dist --single']} />
 
-<Terminal cmd={['$ npx serve dist']} />
+Open [`http://localhost:5000`](http://localhost:5000) to see your project in action. This method is **HTTP only**, so permissions, camera, location, and many other secure features won't work.
 
-Otherwise, use the `--single` flag to redirect all requests to the single **dist/index.html** file.
+</Tab>
 
-<Terminal cmd={['$ npx serve dist --single']} />
+<Tab label="static">
+
+<Terminal cmd={['$ npx serve dist']} />
 
 Open [`http://localhost:5000`](http://localhost:5000) to see your project in action. This method is **HTTP only**, so permissions, camera, location, and many other secure features won't work.
 
 </Tab>
 
+<Tab label="server">
+
+There is no prebuilt solution. You will need to create your own server to serve the static files and API routes.
```

Comment:
This statement feels bad. How can I improve the language here? I tried to get `npx serve` working with this output but I gave up.

@kitten Got any advise?